OneCommunityGlobal/HighestGoodNetworkApp
# Shiwani lint fix for constants

# Description
Lint fix for Constants folder `src/constants/*`

## Main changes explained:
- Removed the folder path from .prettierignore and .eslintignore
- Files that were fixed with lint --fix
- Added export default to timer.js, projects.js, team.js, userProjects.js, materialConstants.js

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as any user
4. check if the app is working properly
5. Check if any of the files in constants folder have any lint errors
6. Additionally, please check if there are any lint errors when you do `npx eslint src/constants/*`
7. Feel free to add any comments or screenshots of your test

## Screenshots:

<img width="635" alt="Screenshot 2023-10-20 at 5 39 12 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78906820/b42a8670-6205-40d7-8576-a7d0bb932092">
